packages/core/src/components/Drawer/Drawer.tsx

```diff
@@ -94,17 +98,14 @@ export const HvDrawer = ({
     ? withTooltip(closeButtonDisplay, buttonTitle, "top")
     : closeButtonDisplay;
 
-  let backgroundColor = checkValidHexColorValue(backgroundColorValue)
-    ? hexToRgbA(backgroundColorValue, 0.8)
-    : backgroundColorValue;
+  const backgroundColor = useRef<string>();
```

Comment:
It seems we are having some problems here (I changed the color to `positive` for DS5):

https://github.com/lumada-design/hv-uikit-react/assets/43220251/8453d233-42ef-4a14-9957-70ef985ecae5

Potential solution:

```suggestion
const [bgColor, setBgColor] = useState<string | undefined>();

  useEffect(() => {
    const getColor = (value?: string) => {
      if (value?.includes("var(--uikit-colors-")) {
        const color = value
          .replace("var(--uikit-colors-", "")
          .replace(")", "")
          .trim();

        return activeTheme?.colors.modes[selectedMode][color];
      } else {
        return value;
      }
    };

    const color = getColor(activeTheme?.drawer.backDropBackgroundColor);

    setBgColor(getBackgroundColor(color));
  }, [selectedMode, activeTheme]);

```